lib/src/embedded_cli.c

```diff
@@ -800,8 +843,8 @@ static void parseCommand(EmbeddedCli *cli) {
 }
 
 static void initInternalBindings(EmbeddedCli *cli) {
-    PREPARE_IMPL(cli)
-
+    PREPARE_IMPL(cli);
+    impl = impl; // gcc unused warning
```

Comment:
Since impl is not used, just remove PREPARE_IMPL(cli)

Reply:
Ah,  Of course.